### ISSUE TITLE
Nautilus-based PDF files blocked by Google chrome macOS

### ISSUE THREAD

**Issue by Popolechien** [openzim/nautilus#96]
Whenever I try to open a PDF on a Nautilus zim-file ([for example this one](https://library.kiwix.org/viewer#zimgit-knots_en/home)) I get an error message `This page has been blocked by Chrome ERR_BLOCKED_BY_CLIENT` 
![Image](https://github.com/user-attachments/assets/06426d6e-59dc-4fbb-b762-8955971a35a4)

This behaviour does not happen on Firefox macOS, nor does it happen on Windows Chrome.
When trying to open PDFs from zimit-generated files (for instance [any of these](https://library.kiwix.org/viewer#survivorlibrary.com_en_all/www.survivorlibrary.com/index.php/Accounting)) I can not reproduce the issue and the PDF opens normally. 

Is this an Apple issue or a Nautilus issue? I am not sure.

**Comment by rgaudin** [comment]
Yeah that 's Chrome specific (and I believe you might have different mac and windows versions of it).

This is triggered by the `target="_blank"` attribute on the link. This was added for #71 but I think it was for _older_ (not that old) iOS Safari.

@kelson42 could you test both links on your iPad?

**Comment by rgaudin** [comment]
@kelson42 no it's not, as I mentioned during the call, the issue expresses itself also on `/content` endpoint, so out of the kiwix-serve iframe.

That said, I can't reproduce #71 anymore as all the devices I have access to are up to date. What's your suggestion?
